### ISSUE TITLE
Flow content not copying over when importing flows.json

### ISSUE THREAD

**Issue by robmarcer** [flowforge/flowforge#1163]
### Current Behavior

When I export a flow from a local Node Red then import it to staging FlowForge the first import seems to work but all subsequent imports do not change the flows.

### Expected Behavior

Each time I import the flow the content on FlowForge should be updated accordingly.

### Steps To Reproduce

https://d.pr/v/fErtWf

### Environment

- FlowForge version: v0.10.1-c1052a9-20221027.0
- Node.js version:
- npm version:
- Platform/OS: MacOS Ventura
- Browser: Latest Chrome

**Comment by robmarcer** [comment]
Re-tested on v1.0.2-64d8bc9-20221028.0, it's slightly worse now as the Node-RED instance doesn't auto restart. It was suggested I try manually restarting it, when I did that it got stuck in a crash loop - https://d.pr/i/dbHXTk

**Comment by robmarcer** [comment]
The reboot loop was resolved by manually stopping and starting the project. Once I'd done that the project started up OK and the flow I'd tried to import was loaded correctly. There are too many variables for me to be confident that this is a bug. I have a hunch that the second time I saw issues it was because the FF version had been updated but I'd not suspended the project. I suggest this is rejected as a bug.
